n3fit/src/n3fit/backends/keras_backend/MetaModel.py

```diff
@@ -46,7 +46,8 @@
 }
 
 NN_PREFIX = "NN"
-PREPROCESSING_PREFIX = "preprocessing_factor"
+NN_LAYER = "NNs"
```

Comment:
Is the fact that `NN_LAYER` is prefixed with `NN` like `NN_PREFIX` a necessary thing? Otherwise it could be a problem in the future.

If it is not necessary, maybe `NN_LAYER` could be `Many_NN` (or I don't know, any other name, it should only be a name).

Reply:
No that shouldn't be necessary, I renamed it to `NN_LAYER_ALL_REPLICAS`, and same for `PREPROCESSING`, is that ok?

Reply:
I think my message was confusing :__

I meant `NN_LAYER = "Many_NN"` so that `NN_LAYER` doesn't use the `NN` prefix! The name of the variable itsel fis ok.

Reply:
Ok how about `all_NNs`?

Reply:
Whatever name not `NN`-prefixed works, if every other module uses this constant it can be changed for something more beautiful at any point.